test/integration/accessibility_spec.js

```diff
@@ -106,4 +106,37 @@ describe("accessibility", () => {
       );
     });
   });
+
+  describe("Annotations order", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("fields_order.pdf", ".annotationLayer");
```

Comment:
I don't think the test will work unless you actually include this PDF in the patch :-P

Reply:
pfff....